sdcm/nemesis.py

```diff
@@ -5246,6 +5246,35 @@ def disrupt_grow_shrink_zero_nodes(self):
         znode = random.choice([node for node in self.cluster.zero_nodes if node.dc_idx == self.target_node.dc_idx])
         self.decommission_nodes(nodes=[znode])
 
+    def disrupt_serial_restart_elected_topology_coordinator(self):
+        """ Rolling restart of topology coordinator node,
+        should trigger new coordinator node election
+        """
+        if not self.target_node.raft.is_consistent_topology_changes_enabled:
+            raise UnsupportedNemesis("Consistent topology changes feature is disabled")
+
+        self.use_nemesis_seed()
+        num_of_restarts = random.randint(1, len(self.cluster.nodes))
+        election_wait_timeout = random.choice([1, 5, 10, 15])
+
+        for _ in range(num_of_restarts):
+            coordinator_node = get_topology_coordinator_node(cluster=self.cluster)
+            self.log.debug("Coordinator node: %s, %s", coordinator_node, coordinator_node.name)
+            if coordinator_node != self.target_node:
+                self.set_current_running_nemesis(coordinator_node)
```

Comment:
I think we shouldn't care if it's target_node or not. Target node should be unset (with `unset_current_running_nemesis`) before loop starts - as it's not being target node anymore.

Reply:
remove it